tests/php/ORM/UnionListTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace SilverStripe\ORM\Tests;
+
+use SilverStripe\Dev\Debug;
+use SilverStripe\ORM\ArrayList;
+use SilverStripe\ORM\UnionList;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\ORM\Tests\DataObjectTest\ValidatedObject;
+
+class UnionListTest extends SapphireTest
+{
+    /**
+     * Borrow the model from DataObjectTest
+     * {@inheritDoc}
+     */
+    protected static $fixture_file = 'DataObjectTest.yml';
```

Comment:
If you remove the fixture but still create stub objects you need to add `protected $usesDatabase = true;` - otherwise it won't create a test DB/table